### PR TITLE
Add controller status to zwave_js device info card

### PR DESCRIPTION
**Pull request by raman325**
## Proposed change
In https://github.com/home-assistant/core/pull/99252 we added support for zwave_js controller status. This PR adds it to the frontend.

NOTE - there's a case to be made here that we actually remove this from the frontend since we have the status sensors available as entities. But I will leave that to the reviewers to decide

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to core pull request: https://github.com/home-assistant/core/pull/99252

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/data/zwave_js.ts [hunk 1]
```diff
@@ -572,6 +589,19 @@ export const unprovisionZwaveSmartStartNode = (
     node_id,
   });
 
+export const subscribeZwaveControllerStatus = (
+  hass: HomeAssistant,
+  device_id: string,
+  callbackFunction: (message: ZWaveJSControllerStatusUpdatedMessage) => void
+): Promise<UnsubscribeFunc> =>
+  hass.connection.subscribeMessage(
+    (message: any) => callbackFunction(message),
```
Comment (bramkragten) [inline src/data/zwave_js.ts:598]: don't use `any` as type here

File: src/panels/config/devices/device-detail/integration-elements/zwave_js/ha-device-info-zwave_js.ts [hunk 2]
```diff
@@ -110,18 +139,22 @@ export class HaDeviceInfoZWaveJS extends SubscribeMixin(LitElement) {
...
             ${this._node.node_id}
           </div>
+          <div>
+            ${this.hass.localize(
+              "ui.panel.config.zwave_js.device_info.node_status"
+            )}:
+            ${this.hass.localize(
+              this._node.is_controller_node
+                ? `ui.panel.config.zwave_js.controller_status.${
+                    controllerStatus[this._controller!.status]
```
Comment (bramkragten) [inline src/panels/config/devices/device-detail/integration-elements/zwave_js/ha-device-info-zwave_js.ts:149]: Use the enum for this

Comment (raman325) [reply src/panels/config/devices/device-detail/integration-elements/zwave_js/ha-device-info-zwave_js.ts:149]: won't that return the enum value not the name?

**Comment by raman325** [comment]
closing this in favor of removing node status instead, see https://github.com/home-assistant/core/pull/99252#discussion_r1309640700

**Pull request closed without merging**